### PR TITLE
added num_reads to configure num shots per task in the qbsolv notebook

### PR DESCRIPTION
**Pull request by jzhzhu**
*Issue #, if available:*
N/A

*Description of changes:*

Added a QBSolv parameter 'num_reads' to allow users to adjust the number of shots per task when use QBSolv().sample_qub(...).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb [hunk 1]
```diff
@@ -154,7 +154,10 @@
     "# define the number of repeats for the hybrid solver to search for the optimal solution.\n",
```
Comment (emkessler) [inline examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb:154]: <pre class="ql-syntax" spellcheck="false">Line #11.    # define the number of shots per task for the hybrid solver to search for the optimal solution.
</pre><p> </p><p>change to</p><p> </p><pre class="ql-syntax" spellcheck="false"># define the number of shots (or reads) per task on the D-Wave QPU
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (emkessler) [inline examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb:154]: <pre class="ql-syntax" spellcheck="false">Line #12.    num_reads = 1000 # default value set by D-wave is 1000 per task
</pre><p> </p><p>could we change the value to 100? Or does that break the example? </p><p> </p><pre class="ql-syntax" spellcheck="false">num_reads = 100 # The default value for D-Wave Advantage_system1 is 1000
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Review by jzhzhu — COMMENTED** [review]
The changes look good.  Tested setting 'num_reads = 100' and it worked for the notebook.

**Review by emkessler — DISMISSED** [review]

File: examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb [hunk 2]
```diff
@@ -168,9 +171,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "conda_braket",
+   "display_name": "Python 3",
```
Comment (christianbmadsen) [inline examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb:174]: This is probably an inadvertent change of meta data from your editor but please revert back to conda_braket

File: examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb [hunk 3]
```diff
@@ -168,9 +171,9 @@
...
    "language": "python",
-   "name": "conda_braket"
+   "name": "python3"
```
Comment (christianbmadsen) [inline examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb:176]: This is probably an inadvertent change of meta data from your editor but please revert back to conda_braket

File: examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb [hunk 4]
```diff
@@ -182,7 +185,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.8.8"
```
Comment (christianbmadsen) [inline examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb:188]: Please change pack to 3.7.7. Thanks!

**Review by christianbmadsen — COMMENTED** [review]
Thank you so much for the contribution. Please address the minor meta data comments and I will approve. You may need to open in a different editor but as long as you look at the raw file the revert of the meta data should be trivial.

**Review by christianbmadsen — APPROVED** [review]
Thank for addressing the comments. The adjusted python version in the meta data is fine. Approving.

**Review by christianbmadsen — APPROVED** [review]

**Pull request merged**